pkg/db/model/vulnerability.go

```diff
@@ -34,22 +35,35 @@ func NewVulnerabilityModel(vulnerability db.Vulnerability) VulnerabilityModel {
 		panic(err)
 	}
 
-	proxy, err := json.Marshal(vulnerability.ProxyVulnerabilities)
+	proxy, err := json.Marshal(vulnerability.RelatedVulnerabilities)
```

Comment:
Why is `proxy` being used here since we are changing from `ProxyVulnerabilities` to `RelatedVulnerabilities`? Can this be named `related` instead? Or is there another reason I'm missing to keep `proxy` around?

Reply:
yeah, I didn't update all of the local variables where "proxy" was used --I'll comb through and look for more instances like this 👍